ComparePaymentProcessors.md

```diff
@@ -0,0 +1,95 @@
+# Payment Processor Comparison
+
+Most new merchants will likely only consider the price of the service. Since BTCPay is free, that may have led you here and if so, welcome. 
+
+We want users to understand not only BTCPay, but also how payments can be processed on behalf of a merchant. In order to help users understand trade-offs when using various cryptocurrency payment processing methods. We don't know which payment processors provide which services. That will need to be part of your own research. The following list is a good place to start:
+
+* [Features](#Features)
+* [Security](#Security)
+* [Privacy](#Privacy)
+* [Decentralized](#Decentralized)
+* [Fiat](#Fiat)
+
+----
+
+## Features
+
+Every payment processor has features, here are some BTCPay features:
+
+* **Bitcoin** - Accepting Bitcoin is the first step.
+* **Altcoins** - Accept cryptocurrency alternatives to Bitcoin.
+* **Lightning** - Rapid Bitcoin microtransactions using the Lightning Network.
+* **Integrations** - Connect to your website using WooCommerce, Drupal, Magneto, Prestashop and custom integrations.
+* **Point Of Sale** - POS Interfaces for physical stores.
+* **Unlimited Stores** - Merchants can process payments for their own stores, or for others.
+* **Merchant Support** - 24/7 open source community support.
+* **Translations** - Customers can pay in 20 different languages. 
+
+----
+
+## Security
+
+First rule of Bitcoin is always keep your private keys *private*. Using a secure wallet is recommended for new merchants as the only provider (creator) of private keys. If there is a chance that someone else (such as a website) knows, stores, or provides your private keys to you, it's generally accepted that they are not actually private. 
+
+Secondly, there is another area of security to consider on the applications layer where you have two main options:
+
+* **Option 1**: Most payment processors (including BTCPay) use the [BIP 21][2] standard.
+* **Option 2**: Others use variations of the [BIP 70][3] standard.
+
+     * **Note**: [BIP 70 has recently been depreciated in Bitcoin Core][1].
+     * Many wallets do not allow payments to BIP 70 invoice urls.
+     * Need help converting obsolete BIP 70 invoice urls to a BIP 21 invoice url? No problem, we can help with that too. <br>Here's our [BIP 70 invoice url converter][4].
+
+----
+
+## Privacy
+
+BTCPay will never ask a merchant for any personal identification other than your email address. 
```

Comment:
This can be confusing. What are you referring to here exactly?
BTCPay never asks for any emails, this sounds like BTCPay is a third party which collects emails.

Reply:
Rather than focusing on email, I would focus:
- BTCPay is private because there's no middleman
- Any information is shared only between the seller and a customer
- If merchant runs a self-hosted BTCPay, that means they are running a full Bitcoin node, which is the ultimate privacy since no middleman can listen to their transactions. This is neat resource https://en.bitcoin.it/wiki/Why_Your_Business_Should_Use_a_Full_Node_to_Accept_Bitcoin
- No address re-use

Reply:
Great ideas, I will add them thanks. The email i was referring to is when merchants log into BTCPay but I agree it is confusing. I will remove that.